app/searchhandler.py

```diff
@@ -166,8 +172,17 @@ async def post(self, request):
             search_criteria_with_range['rangeHigh'] = high_value
             search_criteria_with_range['rangeLow'] = low_value
 
-            retrieve_employee_info = get_employee_records(
-                search_criteria_with_range)
+ 
+            if previous_user_missing_device != False:
+                # if the checkbox is not false, the default value
+            #   raise Exception("Checkbox TICKED" + str( data.get('user_missing_device')   ) )
```

Comment:
These commented-out lines should probably be removed

Reply:
Removed, they were for testing